docs/DeveloperGuide.md

```diff
@@ -505,6 +519,41 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
       Steps 1a1 to 1a2 are repeated until the data entered are correct.  
       Use case resumes from Step 2.
 
+**Use Case: UC09 - Find lessons by student names**  
+**MSS:**
+
+1. Tutor keys in a keyword in the required field, to find lessons with students whose names contain any of the specified keywords.
+1. TutorEase lists lessons that match the requirement.    
+   Use case ends.
+
+**Extensions:**
+
+* **1a.** TutorEase detects bad or wrongly formatted inputs.
+    * **1a1.** TutorEase prompts Tutor with correct format.
+    * **1a2.** Tutor enters new data.  
+      Steps 1a1 to 1a2 are repeated until the data entered are correct.  
+      Use case resumes from Step 2.
+* **1b**. No lessons found matching the entered keyword.  
+    * **1b1**. TutorEase displays an error message, telling Tutor that no lessons have been found with the given keyword.
```

Comment:
maybe it could be
 TutorEase displays an error message, telling the tutor that no lessons have been found with the given keyword.

Reply:
If for consistency, we can use similar to the top one
`TutorEase displays an error message, showing no lessons have been found with the given keyword`